### DISCUSSION TITLE
Icon Request: fa-hairdryer

### DISCUSSION THREAD

**Discussion by jiwon1105** [FortAwesome/Font-Awesome#21177] [Icon Requests]
Hair Dryer
https://www.shareicon.net/hair-dryer-technology-dryer-tools-and-utensils-hairdressing-776638
![technology](https://cloud.githubusercontent.com/assets/27586528/25067198/cc201a2c-2276-11e7-86c8-0b5f23f792af.png)

**Comment by AlexanderTserkovniy** [comment]
Seems like in their heads, no website should be using hair dryer icon in addition to weird namings.
P.S. I also need this icon.

Scanner gun a little bit similar

https://fontawesome.com/icons/scanner-gun?f=sharp&s=light

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
